Cython/Compiler/PyrexTypes.py

```diff
@@ -5515,6 +5517,9 @@ def write_noexcept_performance_hint(pos, env, function_name=None, void_return=Fa
         solutions.append(
             "Use an 'int' return type on %s to allow an error code to be returned." %
             the_function)
+    if is_from_pxd and not void_return:
+        solutions.append(
+            "Declare any exception value explicitly for functions in pxd files.")
```

Comment:
This doesn't seem tested.

Reply:
That's mostly because it's not on the first line of the message and the tests don't currently capture the rest. I'm not sure that's hugely easy to change (although it would probably be possible).

Reply:
Ah. Ok. Well. Not for now then.